src/text-expander-element.ts

```diff
@@ -201,7 +203,8 @@ export default class TextExpanderElement extends HTMLElement {
   connectedCallback() {
     const input = this.querySelector('input[type="text"], textarea')
     if (!(input instanceof HTMLInputElement || input instanceof HTMLTextAreaElement)) return
-    const state = new TextExpander(this, input)
+    const multiWord = this.hasAttribute('multiword')
```

Comment:
`hasAttribute` will return `true` even in the case where the value provided is `false`, right?
https://developer.mozilla.org/en-US/docs/Web/API/Element/hasAttribute

So, if the element is `<text-expander-element multiword="false" />` it will still use the `multiword` behavior, even if this isn't intended, I think.

Reply:
I was thinking of this initially and did consider adding this parsing, but honestly I'm not sure how if it's useful in this case. This is a relatively common thing in `JSX`, but `JSX` is not `HTML`. in the [`spec`](https://html.spec.whatwg.org/#boolean-attributes) it's advised to use the presence/absence of the attribute to denote boolean.
> The values "true" and "false" are not allowed on boolean attributes. To represent a false value, the attribute has to be omitted altogether.

Reply:
@lvivski thanks for the response! That makes sense - since it seems like such an edge case that someone would put `multiword="false"` instead of just omitting, _and_ that it would be pretty easy for them to fix, _and_ that the spec recommends omitting the attribute entirely, I think that just checking the presence is good.

Thanks!